openam-core/src/test/java/org/forgerock/openam/cts/worker/process/CTSWorkerBaseProcessTest.java

```diff
@@ -44,7 +46,7 @@ public class CTSWorkerBaseProcessTest {
     public void setUp() throws Exception {
         mockQuery = mock(CTSWorkerQuery.class);
         mockFilter = mock(CTSWorkerFilter.class);
-        mockProcess = mock(CTSWorkerBaseProcess.class);
+        mockProcess = spy(CTSWorkerBaseProcess.class);
```

Comment:
Is this intentional change?

Reply:
Yes. We need to call the real method in the `shouldUseQueryForNextPage` test case (the mocked variant is not sufficient). The previous implementation worked because of different behaviour of the "PowerMock Mocker" (i think).